src/content/deployment/subctl/_index.en.md

```diff
@@ -196,24 +196,38 @@ that this verification is disruptive.
 
 #### benchmark throughput
 
-`subctl benchmark throughput <kubeconfig1> <kubeconfig2>`
+`subctl benchmark throughput <kubeconfig1> [<kubeconfig2>] [flags]`
 
 The `benchmark throughput` command runs a throughput benchmark test between two specified clusters.
 When running `benchmark throughput` command, two type of tests will be issued:
 
 * Pod to Pod - where both Pods are scheduled on Gateway nodes
 * Pod to Pod - where both Pods are scheduled on non-Gateway nodes
 
+When using the flag `--intra-cluster-baseline`, the following test will be issued:
+
+* Pod to Pod - where both Pods are inside the same cluster, from Non-Gateway to Gateway
+
```

Comment:
Since we document this in the `flags` section  below I think this is redundant. Same on L221 below.